products/pdt/cd/base/configmap.yaml

```diff
@@ -15,3 +15,4 @@ data:
   UNPACK_DIR: "/data/download"
   NR_THREADS: "1"
   BATCH_BLOCKS: "1000"
+  BUFFER_SIZE: "5"
```

Comment:
Refers to the amount of blocks to build the buffer until offloading to bq. 5 is a reasonable currently for ZQ1. Once moving to ZQ2 can be changed to 50